.github/workflows/publish.yml

```
@@ -173,13 +300,11 @@ jobs:
-
-      - name: Pull published image
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          docker pull --platform "${{ matrix.platform }}" \
-            "ghcr.io/${{ github.repository }}/${{ matrix.image }}:${{ needs.prepare.outputs.date }}-${{ github.run_number }}"
+          printf %s "$GITHUB_TOKEN" \
+            | oras login ghcr.io \
+                --username "${{ github.repository_owner }}" \
+                --password-stdin
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)